README.md

```diff
@@ -118,6 +118,80 @@ Example for a subject like `Foo::Bar#baz` it will run all example groups with de
 `Foo::Bar#baz`, `Foo::Bar` and `Foo`. The order is important, so if mutant finds example groups in the
 current prefix level, these example groups *must* kill the mutation.
 
+Reading Mutation Output
+-----------------------
+
+Mutation output is grouped by selection groups. Each group contains three sections:
+
+1. An identifier for the current group.
+
+   **Format**:
+
+   ```text
+   [SUBJECT EXPRESSION]:[SOURCE LOCATION]:[LINENO]
+   ```
+
+   **Example**:
+
+   ```text
+   Book#add_page:Book#add_page:/Users/johnbackus/Downloads/mutant-examples/lib/book.rb:18
```

Comment:
:thumbsup: originally did that since I had the explanation of the identifier.  Didn't want the hash to not match and didn't want to build the examples again